src/ui/public/share/styles/index.less

```diff
@@ -1,21 +1,79 @@
-share-object-url {
-  .input-group {
-    display: flex;
+.share-dropdown {
+  display: flex;
+  padding: 5px 15px;
+}
 
-    .clipboard-button {
-       border-top-left-radius: 0;
-       border-bottom-left-radius: 0;
-    }
+.share-panel {
+  flex: 1 1 0%;
+}
 
-    .shorten-button {
-       border-top-right-radius: 0;
-       border-top-left-radius: 0;
-       border-bottom-right-radius: 0;
-       border-bottom-left-radius: 0;
-    }
+.share-panel--left {
+  margin-right: 30px;
+}
+
+.share-panel--right {
+  margin-left: 30px;
+}
+
+  .share-panel-title {
```

Comment:
Good catch, I will change the name, which will justify the indentation (to indicate the parent-child relationship visually).